### ISSUE TITLE
Files uploaded to trashed folder

### ISSUE THREAD

**Issue by NLLAPPS** [googledrive/android-demos#66]
There seems to be a big delay synchronizing file actions.

Create a folder with your app
Upload a file to your folder with your app
Go to Google Drive web and trash the folder.
Wait 1-2 minutes and try to upload another file in to your folder.
API accepts and uploads it and puts it in the to the folder in the trash.
I tried excluding deleted folder. Even though I used 
`searchFilter.add(Filters.eq(SearchableField.TRASHED, false));` in my search query, API returnred the folder marked as  `metadata.isTrashed()== false`

**Comment by asrivas** [comment]
Are you still experiencing this issue?

**Comment by NLLAPPS** [comment]
Yes. There seems to be other issues with new connectionless API too. Most important one ( [https://github.com/gsuitedevs/android-samples/issues/68](https://github.com/gsuitedevs/android-samples/issues/68) )seems to happen with paid accounts where space is 1TB+ and user has ovr 15GB data,

**Comment by jpoehnelt** [comment]
Closing this issue as the repository is archived.
